include/highfive/bits/H5Node_traits_misc.hpp

```diff
@@ -94,10 +95,15 @@ NodeTraits<Derivate>::getDataSet(const std::string& dataset_name,
 }
 
 template <typename Derivate>
-inline Group NodeTraits<Derivate>::createGroup(const std::string& group_name) {
+inline Group NodeTraits<Derivate>::createGroup(const std::string& group_name,
+                                               bool parents) {
+    RawPropertyList lcpl(Properties::LINK_CREATE);
+    if(parents) {
```

Comment:
```suggestion
    if (parents) {
```

Reply:
I would be helpful to have a clang-format file to make all style consistent (even if clang-format does ugly things sometimes).